searchlib/src/tests/attribute/searchable/attributeblueprint_test.cpp

```diff
@@ -210,5 +228,96 @@ TEST("requireThatFastSearchLocationTermsWork") {
     EXPECT_TRUE(search(node, attribute_manager));
 #endif
 }
+
+AttributeVector::SP
+make_tensor_attribute(const vespalib::string& name, const vespalib::string& tensor_spec)
+{
+    Config cfg(BasicType::TENSOR, CollectionType::SINGLE);
+    cfg.setTensorType(ValueType::from_spec(tensor_spec));
+    return AttributeFactory::createAttribute(name, cfg);
+}
+
+AttributeVector::SP
+make_int_attribute(const vespalib::string& name)
+{
+    Config cfg(BasicType::INT32, CollectionType::SINGLE);
+    return AttributeFactory::createAttribute(name, cfg);
+}
+
+template <typename BlueprintType>
+const BlueprintType*
+as_type(const Blueprint& blueprint)
+{
+    const auto* result = dynamic_cast<const BlueprintType*>(&blueprint);
+    ASSERT_TRUE(result != nullptr);
```

Comment:
if you assert here, why not just return by reference?

Reply:
Fixed